specification/securityinsights/resource-manager/Microsoft.SecurityInsights/stable/2020-01-01/SecurityInsights.json

```diff
@@ -0,0 +1,2489 @@
+          "type": "array"
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "AlertRule": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/ResourceWithEtag"
+        },
+        {
+          "$ref": "#/definitions/AlertRuleKind"
+        }
+      ],
+      "description": "Alert rule.",
+      "discriminator": "kind",
+      "required": [
+        "kind"
+      ],
+      "type": "object"
+    },
+    "AlertRuleKind": {
+      "description": "Describes an Azure resource with kind.",
+      "properties": {
+        "kind": {
+          "description": "The kind of the alert rule",
+          "enum": [
+            "Scheduled",
+            "MicrosoftSecurityIncidentCreation",
+            "Fusion"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "AlertRuleKind",
+            "values": [
+              {
+                "value": "Scheduled"
+              },
+              {
+                "value": "MicrosoftSecurityIncidentCreation"
+              },
+              {
+                "value": "Fusion"
+              }
+            ]
+          }
+        }
+      },
+      "required": [
+        "kind"
+      ],
+      "type": "object"
+    },
+    "AlertRuleTemplate": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        },
+        {
+          "$ref": "#/definitions/AlertRuleKind"
+        }
+      ],
+      "description": "Alert rule template.",
+      "discriminator": "kind",
+      "required": [
+        "kind"
+      ],
+      "type": "object"
+    },
+    "AlertRuleTemplateDataSource": {
+      "description": "alert rule template data sources",
+      "properties": {
+        "connectorId": {
+          "description": "The connector id that provides the following data types",
+          "type": "string"
+        },
+        "dataTypes": {
+          "description": "The data types used by the alert rule template",
+          "items": {
+            "type": "string"
+          },
+          "type": "array"
+        }
+      },
+      "type": "object"
+    },
+    "AlertRuleTemplatePropertiesBase": {
+      "description": "Base alert rule template property bag.",
+      "properties": {
+        "alertRulesCreatedByTemplateCount": {
+          "description": "the number of alert rules that were created by this template",
+          "type": "integer"
+        },
+        "createdDateUTC": {
+          "description": "The time that this alert rule template has been added.",
+          "format": "date-time",
+          "readOnly": true,
+          "type": "string"
+        },
+        "description": {
+          "description": "The description of the alert rule template.",
+          "type": "string"
+        },
+        "displayName": {
+          "description": "The display name for alert rule template.",
+          "type": "string"
+        },
+        "requiredDataConnectors": {
```

Comment:
Property name refers to "connectors" whereas the description has been renamed to "sources". Which is correct, and please can you keep this consistent?

Reply:
Fixed